core-bundle/src/Resources/contao/dca/tl_article.php

```diff
@@ -627,25 +627,20 @@ public function getActiveLayoutSections(Contao\DataContainer $dc)
 			$objPage = Contao\PageModel::findWithDetails($dc->activeRecord->pid);
 
 			// Get the layout sections
-			foreach (array('layout', 'mobileLayout') as $key)
+			if ($objPage->layout)
 			{
-				if (!$objPage->$key)
-				{
-					continue;
-				}
-
-				$objLayout = Contao\LayoutModel::findByPk($objPage->$key);
+				$objLayout = Contao\LayoutModel::findByPk($objPage->layout);
 
 				if ($objLayout === null)
 				{
-					continue;
+					return array();
```

Comment:
Are you sure that this is correct?

Reply:
Yes, it's not a loop anymore so if there's nothing to load the options from, it's going to be an empty array.